scripts/plot_6_taxa_profile.R

```diff
@@ -163,15 +171,16 @@ if(length(unique(metadata_df[[opt$factor]]))>1){
 
 #plot_PCoA_out <- plot_PCoA(distance_lab, profile_phyloseq_ra, title_name, out_name)
 
-plot_PCoA_out <- plot_PCoA(distance_lab, profile_phyloseq_ra, color=opt$factor, label = if (!is.null(opt$time)) opt$time else "sample", shape=opt$factor2)
+plot_PCoA_out <- plot_PCoA(distance_lab, profile_phyloseq_ra, color=opt$factor2, label = if (!is.null(opt$time)) opt$time else "sample", shape=opt$factor)
```

Comment:
Wasn't this a temporary change for the DREAM cohort? Shouldn't we revert it back?